rock.osdeps

```diff
@@ -367,9 +367,8 @@ unzip:
 
 libqglviewer:
     ubuntu:
-        '12.04,12.10,13.04': libqglviewer-qt4-dev
-        '16.04': libqglviewer-dev-qt4
-        default: libqglviewer-dev
+        '12.04,12.10,13.04,13.10,14.04,14.10,15.04,15.10': libqglviewer-qt4-dev
```

Comment:
You sure about that one ? The previous default was 'libqglviewer-dev' without the qt4